app/controllers/users_controller.rb

```diff
@@ -0,0 +1,22 @@
+class UsersController < ApplicationController
+  before_action :authenticate_user!
+  def index
```

Comment:
@apeksha2697  add one line spacing between before_action and index

Reply:
done